package.json

```diff
@@ -22,6 +22,7 @@
     "start": "start-storybook -p 9001 -c storybook -s ./storybook/static",
     "build": "lerna run --parallel build -- --env production",
     "build:storybook": "build-storybook -c storybook -o ./storybook/public -s ./storybook/static",
+    "bump": "yarn changeset bump",
```

Comment:
Let's omit and just use `yarn changeset bump` directly and we'll just document it in the README.

Reply:
:+1